python-client/wmill/wmill/client.py

```diff
@@ -81,21 +87,71 @@ def run_script_async(
     ).content.decode("us-ascii")
 
 
-def run_script_sync(hash: str, args: Dict[str, Any] = {}, verbose: bool = False) -> Dict[str, Any]:
+def run_script_sync(
+    hash: str, 
+    args: Optional[Dict[str, Any]] = None, 
+    verbose: bool = False,
+    assert_result_is_not_none: bool = True,
+    cancel_atexit: bool = True,
```

Comment:
cancel_at_exit*

I had a hard time reading atexit :)

Reply:
I'll rename it :)